src/components/Editor.jsx

```diff
@@ -142,6 +152,7 @@ class Editor extends React.Component {
 }
 
 Editor.propTypes = {
+  editorTextSizeIsLarge: PropTypes.bool.isRequired,
```

Comment:
Since this is the `Editor` component, we can probably just call this prop `textSizeIsLarge`